Kitodo/src/main/java/de/sub/goobi/export/dms/ExportDms.java

```diff
@@ -255,17 +228,59 @@ public boolean startExport(Process process, URI inZielVerzeichnis, DigitalDocume
             exportDmsTask.setProgress(1);
         }
 
+        boolean downloadImages = downloadImages(process, userHome, zielVerzeichnis);
+        if (!downloadImages) {
+            return false;
+        }
+
         /*
-         * der eigentliche Download der Images
+         * zum Schluss Datei an gewünschten Ort exportieren entweder direkt in den
+         * Import-Ordner oder ins Benutzerhome anschliessend den Import-Thread starten
          */
+        if (process.getProject().isUseDmsImport()) {
+            exportWithImport(process, gdzfile, userHome);
+        } else {
+            exportWithoutImport(process, gdzfile, userHome);
```

Comment:
New method naming is bad. This is a synchronous export and running in front of the user. Until this export is not finished user may should or could not do anything else.